Furball.Vixie.Backends.Shared/Furball.Vixie.Backends.Shared.csproj

```
@@ -8,8 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="FontStashSharp" Version="1.2.0" />
-        <PackageReference Include="ImGui.NET" Version="1.87.3" />
+        <PackageReference Include="FontStashSharp" Version="1.1.8" />
```

This should be updated to this
```suggestion
        <PackageReference Include="FontStashSharp" Version="1.2.0" />
```

---

I mean I don't think this changed in my PR, but sure